### ISSUE TITLE
Problems with installation

### ISSUE THREAD

**Issue by bittelli** [gprMax/gprMax#300]
Hello
I am installing gprMax on Windows 10. During the "python setup.py build" I get an error while compiling with Visual Studio

building 'gprMax.cython.fields_updates_dispersive' extension

gprMax\cython\fields_updates_dispersive.c(1181): error C2061: errore di sintassi: identificatore '__pyx_t_double_complex'
....
... many of these and it ends with

error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2

I checked and the pathway listed above is in the system environment variables. 

Thank you,

Marco

**Comment by Kazumi777** [comment]
Well, i also have this problem, but i found that the author had updated something 16 days ago. So i try to install the previous release(v.3.1.5 (Big Smoke) at 17 Apr 2019). Surprisingly, I succeeded！

**Comment by bittelli** [comment]
Thank you Kazumi, I will try and let you know. 
Marco

**Comment by bittelli** [comment]
Thank you Kazumi !!, I installed version 3.1.5 and it works. Marco

**Comment by craig-warren** [comment]
@bittelli @Kazumi777 can you provide some more information about what machine you are using and the operating system so we can look into this further please.

**Comment by bittelli** [comment]
Hello Prof Warren,
I have Windows 10, 64 bit x64,  Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz   1.99 GHz.
The installation procedure determines the errors listed above, with correct pathway for C compiler (Visual Studio) for Cython.
I followed the instructions listed in the website. 

Thank you for your assistance,

Marco

**Comment by neilwang0913** [comment]
I have the same issue and change 3 machines (all give the same error) Is there any bug related to the VS2019?

**Comment by ElmoreJames83** [comment]
I have ran into the same Issue and I solved it like @Kazumi777 by downloading the main commit from 17 Apr 2019.

I compiled using Visual Tools 2015, I didn't try 2019.

I am attaching the error log. 

My machine is:
Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz
Windows 10.0.19044 compilation19044

I attach the error log.
[error log 3.txt](https://github.com/gprMax/gprMax/files/7855334/error.log.3.txt)
